CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+**Features**:
+
+- Add `user.id` and `user.segment` to sampling rules. ([#959](https://github.com/getsentry/relay/pull/959))
```

Comment:
With all changes related to dynamic sampling, let's please keep them in "Internal", as this is still under development and not an available feature yet.

Reply:
ok